src/test/java/com/izmaylov/hw15/practiceSlide20/MyListTest.java

```diff
@@ -0,0 +1,62 @@
+package com.izmaylov.hw15.practiceSlide20;
+
+import org.junit.Assert;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.NoSuchElementException;
+
+class MyListTest {
+
+    @Test
+    void shouldThrowExceptionWhenTryingToAddNullValue(){
+        MyList myList = new MyList();
+        String temp = null;
+
+        Assert.assertThrows(NullPointerException.class, () -> myList.add(temp));
+    }
+
+    @Test
+    void shouldIncreaseSizeByOneWhenTryingToAddValue(){
+        MyList myList = new MyList();
+        String temp = "1";
+        int expected = 1;
+        myList.add(temp);
+
+
+        Assert.assertEquals(expected, myList.size);
```

Comment:
If you are using the 5th version of the library, then you must use the checks from it: Assertions